src/cargo/ops/cargo_compile/unit_generator.rs

```diff
@@ -267,14 +268,36 @@ impl<'a> UnitGenerator<'a, '_> {
                 })
                 .collect::<Vec<_>>();
             let suggestion = closest_msg(target_name, targets.iter(), |t| t.name(), "target");
+            let targets_elsewhere = self.get_targets_from_other_packages(filter)?;
+            let need_append_targets_elsewhere = !targets_elsewhere.is_empty();
+            let append_targets_elsewhere = |msg: &mut String| {
+                for (package, targets) in targets_elsewhere {
+                    if !targets.is_empty() {
+                        writeln!(msg, "help: Available {target_desc} in `{package}` package:")?;
```

Comment:
Sorry I missed this but `Available` should be lower case, see https://rustc-dev-guide.rust-lang.org/diagnostics.html#diagnostic-output-style-guide

Reply:
Except we already use `Available`, so we can skip this for now.